includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -313,6 +313,30 @@ public function update_intention(
 		return $this->deserialize_intention_object_from_array( $response_array );
 	}
 
+	/**
+	 * Updates an intention's metadata.
+	 * Unlike `update_intention`, this method allows to update metadata without
+	 * requiring amount, currency, and other mandatory params to be present.
+	 *
+	 * @param string $intention_id - The ID of the intention to update.
+	 * @param array  $metadata     - Meta data values to be sent along with payment intent creation.
+	 *
+	 * @return WC_Payments_API_Intention
+	 * @throws API_Exception - Exception thrown on intention creation failure.
+	 */
+	public function update_intention_metadata(
```

Comment:
Can we make the method signature a one-liner?

Reply:
Done ✅